public/kibi_timeline.js

```diff
@@ -149,25 +149,29 @@ define(function (require) {
         const groupId = group.id;
         const groupColor = group.color;
 
+        searchSource.highlight({
+          fields: {
+            '*': {
+              fragment_size: 0,
+              number_of_fragments: 100
```

Comment:
100 seems to be large for the `number_of_fragments`. Could this be parameterized ?